### ISSUE TITLE
Have autobuilds on the release page.

### ISSUE THREAD

**Issue by OpenRift412** [chocolate-doom/chocolate-doom#1622]
Given how there hasn't been like a full release of Choco on the releases page since 2017 (3.0.1 isn't even there for some reason), I think it would be wise to have the latest autobuilds for Choco show up there (I don't know how setting that up works but I've seen other repos like NBlood and Xash3D do it) or at the very least, have a link to http://latest.chocolate-doom.org/ in the readme.

**Comment by AXDOOMER** [comment]
I think **this project is due for a new release**, just because of the sheer amount of commit that were made. (Yes I know, there's already an [issue](https://github.com/chocolate-doom/chocolate-doom/issues/1369) about that, but I wanted to mention it again)

**Comment by OpenRift412** [comment]
> I think **this project is due for a new release**, just because of the sheer amount of commit that were made. (Yes I know, there's already an [issue](https://github.com/chocolate-doom/chocolate-doom/issues/1369) about that, but I wanted to mention it again)

I agree, though I don't really know how willing they are to actually do regular releases anymore, so this would probably be the best alternative.

**Comment by mfrancis95** [comment]
I just created a release for 3.0.1: https://github.com/chocolate-doom/chocolate-doom/releases/tag/chocolate-doom-3.0.1. Please take a look.

**Comment by OpenRift412** [comment]
> I just created a release for 3.0.1: https://github.com/chocolate-doom/chocolate-doom/releases/tag/chocolate-doom-3.0.1. Please take a look.

That's good, but I think it still is a rather dated release since it still uses midiproc and doesn't even have the updated logos.

**Comment by mfrancis95** [comment]
Starting this here: https://github.com/chocolate-doom/chocolate-doom/pull/1623. Some collaboration from some of the recent and more active contributors would be appreciated.
